components/list/list.css

```diff
@@ -0,0 +1,28 @@
+@import '../../utility/common.css';
+@import '../../installation/styles.css';
+
+/* Unordered list */
+.list--circle {
+  list-style-type: circle;
+}
+.list--square {
+  list-style-type: square;
+}
+
+/* Ordered list */
+.list--roman {
+  list-style-type: upper-roman;
+}
+.list--alpha {
+  list-style-type: lower-alpha;
+}
+
+/* Stacked list */
+.stack {
+  list-style: none;
+}
+.stack__card {
+  box-shadow: 0 0.5em 1em -0.125em rgb(10 10 10 / 10%),
+    0 0 0 1px rgba(129, 113, 113, 0.02);
+    padding: 1em;
+}
```

Comment:
- use `rems`
- try making the box shadow as a variable if it is being reused

Reply:
done.